templates/.vscode/tasks.json

```diff
@@ -0,0 +1,57 @@
+{
+  "version": "2.0.0",
+  "problemMatcher": "$tsc-watch",
+  "tasks": [
+    {
+      "label": "Compile",
```

Comment:
Added a `Compile` task that can be triggered using the VSCode tasks UI. This runs the `prepack` script that's shipped as part of the plugin generator code. If errors are found they will be parsed and showed in VSCode's `Problems View` 

![Screen Shot 2019-07-16 at 3 01 50 PM](https://user-images.githubusercontent.com/1041842/65047532-7613e000-d917-11e9-80cf-08804e6aacfb.png)